src/material/badge/badge.ts

```diff
@@ -84,22 +85,20 @@ export class MatBadge extends _MatBadgeBase implements OnDestroy, OnChanges, Can
   @Input('matBadgePosition') position: MatBadgePosition = 'above after';
 
   /** The content for the badge */
-  @Input('matBadge') content: string | number | undefined | null;
+  @Input('matBadge')
+  get content(): string | number | undefined | null {
```

Comment:
We could pull the `string | number | undefined | null` into a type so we don't have to repeat it in 3 places.

Reply:
Ack, I'm going to leave as-is for now.